dts/arm/nxp/nxp_rt1170_cm7.dtsi

```diff
@@ -45,24 +45,41 @@
 		 * M7 uses different addresses from the M4 core for GPIO2 and
 		 * GPIO3, see pg. 1460 of RT1170 ref manual for example
 		 */
-		gpio2: gpio@42008000 {
+		gpio2: gpio@40130000 {
 			compatible = "nxp,imx-gpio";
-			reg = <0x42008000 0x4000>;
+			reg = <0x40130000 0x4000>;
 			interrupts = <102 0>, <103 0>;
 			label = "GPIO_2";
 			gpio-controller;
 			#gpio-cells = <2>;
 		};
 
-		gpio3: gpio@4200c000 {
+		gpio3: gpio@40134000 {
 			compatible = "nxp,imx-gpio";
-			reg = <0x4200c000 0x4000>;
+			reg = <0x40134000 0x4000>;
 			interrupts = <104 0>, <105 0>;
 			label = "GPIO_3";
 			gpio-controller;
 			#gpio-cells = <2>;
 		};
 
+		fgpio2: gpio@42008000 {
+			compatible = "nxp,imx-gpio";
+			reg = <0x42008000 0x4000>;
+			interrupts = <99 0>;
+			label = "FGPIO_2";
+			gpio-controller;
+			#gpio-cells = <2>;
+		};
+
+		fgpio3: gpio@4200c000 {
+			compatible = "nxp,imx-gpio";
+			reg = <0x4200c000 0x4000>;
+			label = "FGPIO_3";
```

Comment:
Please remove all the "label" properties from this file. There is an active effort to deprecate this from Zephyr.

Reply:
done